packages/android-udev-rules/PKGBUILD

```diff
@@ -2,26 +2,19 @@
 # See COPYING for license details.
 
 pkgname=android-udev-rules
-pkgver=407.b3702da
+pkgver=20200613
```

Comment:
I'm not sure we should move from a git version to a release version. It's quite the contrary we try to do.

Reply:
Use git version as @noraj said.